### DISCUSSION TITLE
3D-IBEX?

### DISCUSSION THREAD

**Discussion by blin100** [IBEXImagingCommunity/ibex_imaging_knowledge_base#37] [Q&A] [answered]
Hi, I was pointed to this github community through the HCA slack, and I had a question based on that message--I can't find any full publication for 3D-IBEX? And I'm curious how the problem of bubbles is solved in a 3D chunk of tissue for this. All I could find was the abstract here:
https://journals.aai.org/jimmunol/article/208/1_Supplement/116.23/236422/3D-IBEX-Achieving-multiplex-3-dimensional-imaging
(Also, has anyone done IBEX on organoids? I'm worried the LiBH would obliterate it with bubbles...

Any help would be appreciated!

**Comment by arroyomejiasaj** [comment, marked as answer]
Hello! Thank you for your question.
 
3D-IBEX, now called Ce3D-IBEX, has not been published yet. But to address your questions, the bubble formation seems to be specific to the type of tissue you work with. We have typically seen their formation in lung 3D sections but not in other tissues. To prevent their formation, we do a couple of things: 
 

1. Prepare the LiBH4 (chemical bleaching solution) and let it rest for 1 hour. When you prepare the LiBH4 it starts forming bubbles in solution, but if you let it rest for 1 hour it eventually stops their formation.
2. We wash the tissue with Molecular Biology Water before adding the LiBH4, we have seen it reduces bubble formation in tissue. 
3. After chemically bleaching the tissue for one hour, we must thoroughly wash out any LiBH4 leftovers. So, we usually do a long wash with multiple changes of washing buffer to wash off any bubbles that may have been formed.
4. We have also been implementing other bleaching options. We have started to use photobleaching as opposed to chemical bleaching. Where we basically expose the tissue to a powerful source of light to get rid of the fluorophore’s signals. As we do not use any LiBH4, no bubbles are formed.

Regarding the organoid Ce3D-IBEX.
It has been done for **Ce3D** imaging but not for Ce3D-IBEX imaging. 

Ce3D imaging (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5584454/) however, the organoid data/images were not published. 

For Ce3D-IBEX to work, ideally, the tissue should be embedded in agarose or fixed to a slide as it makes it easier to handle and treat. It is important that we do not affect the tissue between cycles as this may affect 3D registration. So the tissue must remain virtually the same in composition and position for imaging after every cycle. Although we have done some free-floating tissues, they have been sturdy compared to an organoid.

I hope this helps and if you have more questions send them our way!

**Reply by radtkea** [reply]
This is so exciting to see our first question, thank you blin100, and answer from Armando! Yeah team.

**Reply by natape** [reply]
Amazing!🧡

**Comment by blin100** [comment]
Thanks Armando!

Good luck on publishing the Ce3D-IBEX, and look forward to seeing it in print!
Interesting that the bubbles are tissue specific, must be some sort of enzyme that is found in it? 

I noted that in the original protocol, it recommends waiting some time after dissolving the LiBH4 for the bubbles, but found that efficiency of bleaching seemed to drop off at a similar time? Not a chemist, so I perhaps naively thought that the bubbles were forming at the cost of bleaching activity. Now that I'm looking into it, I'm interested whether you've tried using degassed, distilled/milliq water to reduce the hydrogen gas formation?

For the embedded tissue in agarose, you don't have problems with gas bubbles forming within the agarose? The main reason why I was worried about the organoid staining is that ours are very fragile, and weakly attached to a glass slide--we're still working on how to more stably mount or prepare it, as they aren't really big enough to use the standard sedimentation method. Ours also have neuronal outgrowths from the main body that attach to the glass weakly. Definitely a specialty problem here. Was thinking after your comment of overlaying low melt agarose on top of it just to hold it in place...

Thanks for the answers!
Brian

**Reply by colinchu72** [reply]
Hi Brian,

I tested the 3D-IBEX approach on some human retinal organoids when we were optimising. We embedded in agarose and then cut 300um vibratome sections. It worked pretty well and the bubbles weren't a big problem. A quick wash with water before bleaching reduced bubble formation in my opinion. I didn't adhere them to a slide though and they were free-floating so that may be a difference. It makes re-alignment more challenging, but bubbles float off the sections.

Good luck!

Colin

**Reply by blin100** [reply]
Excellent news, thanks! We'll have to give it a try, the amount of bubbles I got with airway tissue was making me feel like this would be impossible because things would be torn to shreds, but it seems like that was actually a tissue-specific problem (kind of interesting...).

Cheers,
Brian
